cameras/paths.csv

```diff
@@ -36,3 +36,5 @@ Grandstream,camera,GXV3674_[F]HD_VF,(00:0B:82),rtsp://{{username}}:{{password}}@
 Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}}.sdp,H.264,554,live;live2;liveany,,Stream 1;Stream 2;Stream 3,,live2,live,,,,http://download.vivotek.com/downloadfile/downloads/usersmanuals/cc8130manual_en.pdf
 Dahua,camera,HFW81200,(4C:11:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}},H.264,554,,,,,,,admin,admin,,http://www1.dahuasecurity.com/products/ipc-hfw81200e-z-1159.html
 WatchNet,nvr,WatchNet DVR,(52:54:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16,High;Low,Channel {{channel}},1,0,admin,1234,,http://watchnetinc.com/public/pdfs/VMS%20Lite%20Users%20Manual_Ver1.0.pdf
+Luma,nvr,Lum-500/501 Series,(00:0F:0D),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,main stream;sub stream,Channel {{channel}},01,02,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/Luma-500%20Technical%20Reference%20Manual.pdf
```

Comment:
see my issue comment that the actual port is the concatenation of a base port and the channel.